ocaml/xapi/cert_distrib.ml

```diff
@@ -479,6 +479,34 @@ let exchange_certificates_with_joiner ~__context ~uuid ~certificate =
   Worker.local_write_cert_fs ~__context HostPoolCertificate Merge
     [joiner_certificate] ;
   Worker.local_regen_bundle ~__context ;
+  let () =
+    (* now that the primary host trusts the joiner, perform best effort
+     * distribution to remaining hosts (this is not strictly necessary,
+     * as the 'am i missing certs thread' should pick up missing certs) *)
+    let secondary_hosts = Xapi_pool_helpers.get_slaves_list ~__context in
```

Comment:
There is an API to sync config files (currently `/etc/shadow`), could that be reused for syncing the certificates?

Reply:
in theory? (though my opinion is that at this point reimplementing the code we already have for distributing certificates to use `config_file_sync.ml` is creating unnecessary work for ourselves)